docs/FaceDetectionScriptExample.md

```diff
@@ -0,0 +1,128 @@
+# Example of Face Detection with OpenVINO&trade; Model Server<a name="facedetection"></a>
+
+This page guides you through an example script to run face detection inference with OpenVINO&trade; Model Server.
+
+
+- The script [face_detection.py](../example_client/face_detection.py) runs face detection inference requests for all the images
+saved in `image_input_dir` directory. 
+
+- The script can adjust the input image size and change the batch size in the request. It demonstrates how to use
+the functionality of dynamic shape in OpenVINO Model Server and how to process the output from the server.
+
+- The example relies on the model [face_detection_0004](https://github.com/openvinotoolkit/open_model_zoo/blob/master/models/intel/face-detection-retail-0004/description/face-detection-retail-0004.md).
+
+### Running the Script
+
+```bash
+python face_detection.py --help
+usage: face_detection.py [-h] [--input_images_dir INPUT_IMAGES_DIR]
+                         [--output_dir OUTPUT_DIR] [--batch_size BATCH_SIZE]
+                         [--width WIDTH] [--height HEIGHT]
+                         [--grpc_address GRPC_ADDRESS] [--grpc_port GRPC_PORT] [--model_name]
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --input_images_dir INPUT_IMAGES_DIR
+                        Directory with input images
+  --output_dir OUTPUT_DIR
+                        Directory for staring images with detection results
+  --batch_size BATCH_SIZE
+                        how many images should be grouped in one batch
+  --width WIDTH         how the input image width should be resized in pixels
+  --height HEIGHT       how the input height should be resized in pixels
+  --grpc_address GRPC_ADDRESS
+                        Specify url to grpc service. default:localhost
+  --grpc_port GRPC_PORT
+                        Specify port to grpc service. default: 9000
+  --model_name MODEL_NAME
+                        Specify the model name 
+```
+
+### Usage example
+
+Start the OVMS service locally:
+
+```bash
+curl --create-dirs https://download.01.org/opencv/2021/openvinotoolkit/2021.1/open_model_zoo/models_bin/1/face-detection-retail-0004/FP32/face-detection-retail-0004.xml https://download.01.org/opencv/2021/openvinotoolkit/2021.1/open_model_zoo/models_bin/1/face-detection-retail-0004/FP32/face-detection-retail-0004.bin -o model/1/face-detection-retail-0004.xml -o model/1/face-detection-retail-0004.bin
+docker run -d -v $(pwd)/model:/models -p 9000:9000 openvino/model_server:latest  --model_path /models --model_name face-detection --port 9000  --shape auto
```

Comment:
[2020-10-22 21:47:38.396] [serving] [error] Couldn't list directories in path: /models
[2020-10-22 21:47:38.396] [serving] [error] Couldn't start model manager
[2020-10-22 21:47:38.396] [serving] [error] ovms::ModelManager::Start() Error: The provided base path is invalid or doesn't exists